tests/test_import.py

```diff
@@ -1,12 +1,15 @@
 import pyhf
+from pyhf.tensor.numpy_backend import numpy_backend
 import pyhf.readxml
 import json
 import pytest
 import numpy as np
 
+
 def test_import_prepHistFactory():
+    pyhf.set_backend(numpy_backend())
```

Comment:
@kratsg Given what we discussed in chat RE: a03edbdfc9d8308c42e82b12e53d3bc450441be5 do you want me to keep these in? Or do you want me to wait until you add in something that universally protects against this and then rebase against that and remove these `pyhf.set_backend(numpy_backend())` calls?